samples/nrf9160/memfault/README.rst

```diff
@@ -20,7 +20,7 @@ The sample supports the following development kits:
    :header: heading
    :rows: thingy91_nrf9160_ns, nrf9160dk_nrf9160_ns
 
-Before using the Memfault platform, you must register an account in `Memfault`_ and a set up a project according to the instructions in `Memfault documentation <Memfault Docs_>`_.
+Before using the Memfault platform, you must register an account in `Memfault`_ and a set up a project according to the instructions in the `Memfault nRF Connect SDK integration guide`_.
```

Comment:
When refering to creating a new account, when can link to this page that is specifically set up for onboarding Nordic customers: https://goto.memfault.com/create-key/nrf

Reply:
I see there's already a link to a Nordic registration page in links.txt: https://app.memfault.com/register-nordic
Which one is preferred?

Reply:
The one I originally wrote redirects to the one you have there (I was too slow to update my comment), so you can use the existing one